src/ray/stats/metric_defs.h

```diff
@@ -81,6 +81,27 @@ DECLARE_stats(spill_manager_objects_bytes);
 DECLARE_stats(spill_manager_request_total);
 DECLARE_stats(spill_manager_throughput_mb);
 
+///
+/// Plasma Store Metrics
+///
+
+/// Object Lifecycle Manager.
+/// TODO(sang): Specify source for each of metric.
+DECLARE_stats(plasma_local_spillable_objects);
```

Comment:
I initially considered plasma_objects with label (State, Source), but it seems like using {Spillable, evictable, in_use, unsealed} as "state" label is wrong because they are not mutually exclusive. For example, spillable objects are also in_use, so the sum of all labels won't have any meaning.

There are 3 alternatives;
- The current style.
- Current style + add "Source" labvel for each metric. I am in favor of this, but this requires a quite amount of code change, so if you think it is necessary, I will do it in a follow up. 
- plasma object, ("IsSealed", "IsSpillable", "IsEvictable", "IsInUse", "Source"). This will probably create too many labels (total 4 * 16 variations) and also complex to implement, so I think ROI is low.

Also open to hear other alternatives.

Reply:
what's in the source labels? the current style seems fine to me

Reply:
You can have overlapping labels still, by having multiple labels. For example:

{State, IsSpillable}

Reply:
Source is how the object is created (I am using the same term as what's used in flat buffer)

```
DEFINE_stats(plasma_local_objects,
             "The total number of objects broken per source. { Priamry, Transferred, "
             "Error, Restored }",
```